pkg/cloud/vsphere/services/govmomi/create.go

```diff
@@ -71,7 +71,13 @@ func Create(ctx *context.MachineContext, bootstrapToken string) error {
 	}
 
 	if ctx.Session.IsVC() {
-		return vcenter.Clone(ctx, userData)
+		err = vcenter.Clone(ctx, userData)
```

Comment:
Hi @pickledrick,

Please change this to `if err := vcenter.Clone` to reduce the scope of `err`.